src/App.jsx

```diff
@@ -8,37 +9,15 @@ export const firstArticle = {
```

Comment:
The date prop is missing in the firstArticle object. Please add the date property with a valid value.